test/objects_test.go

```diff
@@ -32,7 +32,8 @@ func TestObjectEngine(t *testing.T) {
 	}
 	require.NoError(t, Client.Create(ctx, owner, client.FieldOwner(fieldOwner)))
 	t.Cleanup(func() {
-		if err := Client.Delete(ctx, owner); err != nil {
+		//nolint:usetesting
+		if err := Client.Delete(context.Background(), owner); err != nil {
```

Comment:
What - why does the linter complain here? `t.Context` is canceled _before_ cleanup functions run.

:confused-josh:

https://pkg.go.dev/testing#T.Cleanup
https://pkg.go.dev/testing#T.Context